apps/src/templates/sectionAssessments/MatchAssessmentsOverviewContainer.jsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
```

Comment:
You can also do `import React, {Component, PropTypes} from 'react';` to combine these 2 lines, but I don't there's a preference for one or the other.

Reply:
@Erin007 this was true in the past, but proptypes is supposed to be a separate import now.